tests/tracer/test_settings.py

```diff
@@ -218,7 +218,7 @@ def test_environment_analytics_enabled(self):
             self.assertEqual(config.foo.analytics_sample_rate, 0.5)
 
     def test_analytics_enabled_attribute(self):
-        """" Confirm environment variable and kwargs are handled properly """
+        """ " Confirm environment variable and kwargs are handled properly"""
```

Comment:
```suggestion
        """ Confirm environment variable and kwargs are handled properly"""
```

lol I guess we can get rid of these